models/topics.models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query(`SELECT * FROM topics;`).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "Not found" });
```

Comment:
Line 6 would work fine if we were requesting by ID because we could be searching where ID = "48483939", but in this case if there were no topics we would just want the empty array here not a rejection, nothing has gone wrong.